frontend/public/components/utils/units.js

```diff
@@ -254,4 +258,40 @@ validate.memory = (value='') => {
   return validateNumber(number) || validateMemUnit(unit);
 };
 
+// Convert k8s compute resources values to a base value for comparison.
+// If the value has no unit, it just returns the number, so this function
+// can be used for any quota resource (resource counts). `units.dehumanize`
+// is problematic for comparing quota resources because you need to know
+// what unit you're dealing with already (e.g. decimal vs binary). Returns
+// null if value isn't recognized as valid.
+export const convertToBaseValue = (value) => {
+  if (!_.isString(value)) {
+    return null;
+  }
 
+  const [number, unit] = validate.split(value);
+  const validationError = validateNumber(number);
+  if (validationError) {
+    return null;
+  }
+
+  if (!unit) {
+    return number;
+  }
+
+  // Handle CPU millicores specifically.
+  if (unit === 'm') {
```

Comment:
`M` is megabyte, and "numeric" values aren't a real thing in k8s. This only handles actual k8s compute resources, so `m` is unambiguous.

I'd like to look at removing the numeric stuff. That is really confusing when looking at memory in graphs, and we've gotten bugs on it. (It will say `B` for billion, but it looks like bytes.)